app/pandas.ipynb

```diff
@@ -482,9 +631,7 @@
    "cell_type": "code",
    "execution_count": null,
    "outputs": [],
-   "source": [
-    "# get length of each word"
-   ],
+   "source": "s_length = s.str.len()",
```

Comment:
You have correctly calculated the length of each word. Ensure you print the result to verify the output.